src/app/pages/waiting-room-to-car/components/vehicle-registration/vehicle-registration.ts

```diff
@@ -25,22 +26,42 @@ export class VehicleRegistrationComponent implements OnChanges {
   @Output()
   vehicleRegistrationChange = new EventEmitter<string>();
 
+  @Output()
+  motStatusChange = new EventEmitter<string>();
+
   formControl: UntypedFormControl;
 
   readonly registrationNumberValidator: FieldValidators = getRegistrationNumberValidator();
 
+  constructor(
+    private vehicleProvider: VehicleDetailsApiService,
+  ) {
+  }
+
   ngOnChanges(): void {
     if (!this.formControl) {
       this.formControl = new UntypedFormControl(null, [Validators.required]);
       this.formGroup.addControl('vehicleRegistration', this.formControl);
+      if (this.vehicleRegistration != null) this.getMotAndTax(this.vehicleRegistration);
```

Comment:
Are we doing this every time the vehicleRegistration changes?

Should we add guards around it for example if valid and on blur to avoid excessive calls?

Reply:
no, this will only fire once on page entry and only then if the reg has been previously set.  it is called on an blur action when entered on this page.